harvard-lts/mps-viewer
# Add Mirador Analytics Plugin to Viewer

**Add Mirador Analytics Plugin to Viewer**
* * *

**JIRA Ticket**: [LTSVIEWER-53](https://jira.huit.harvard.edu/browse/LTSVIEWER-53)

* Other Relevant Links: [PR for Mirador Analytics Plugin](https://github.com/harvard-lts/mirador-analytics-plugin/pull/1)

# What does this Pull Request do?
Imports the new mirador-analytics-plugin ([npm package](https://www.npmjs.com/package/@harvard-lts/mirador-analytics-plugin?activeTab=code) and [code repo](https://github.com/harvard-lts/mirador-analytics-plugin)) to add Harvard's analytics tracking code into viewer

# How should this be tested?

A description of what steps someone could take to:
* Rebuild container from this branch (you should see a new package added to the package.json, `@harvard-lts/mirador-analytics-plugin: ^0.2.0`
* If you inspect the viewer, you should see analytics code inserted into two areas of the page, at the top of the `<head>` and the top of the `<body>`
* Check that the tracking id in both places matches this: `GTM-WSHSDQ5`
* If you have access to Harvard Library's account in Google Tag Manager, you can also preview the local environment to ensure that things are firing correctly ([Video for instructions: Check that analytics is firing in Curiosity QA](https://drive.google.com/file/d/1wB3iJphfdrj28KLRgw456cnKeMNcqKb2/view?usp=sharing))

Note: once we have a domain name, this will be added to GA4 for cross domain tracking

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl @enriquediaz
